spec/searchers/responses_searcher_spec.rb

```diff
@@ -73,6 +105,54 @@
     it "should return nothing for non-existent group" do
       expect(search(%(group:norble))).to be_empty
     end
+
+    it "should parse searcher props" do
+      expect(searcher(%(group:"fun group"))).to have_filter_data(groups: items_by_name_and_id(group), advanced_text: "")
+      expect(searcher(%(group:foo))).to have_filter_data(groups: [], advanced_text: "group:(foo)")
+      expect(searcher(%(group:foo source:x))).to have_filter_data(groups: [], advanced_text: "group:(foo) source:(x)")
+    end
+  end
+
+  describe "question qualifier" do
+    let!(:opt_set) { create(:option_set, option_names: :multilevel) }
+    let!(:q1) { create(:question, qtype_name: "long_text", code: "blue", add_to_form: form) }
+    let!(:q2) { create(:question, qtype_name: "long_text", code: "Green", add_to_form: form) }
+    let!(:q3) do
+      create(:question, qtype_name: "select_one", code: "red", option_set: opt_set, add_to_form: form)
+    end
```

Comment:
The more conventional way to set this up is like this:

```
let(:form) { create(:form, question_types: %w[long_text long_text multilevel_select_one]) }
let(:qing1) { form.c[0] }
let(:node3) { form.c[2].option_set.c[0] } 
``` 

Note the second line may not be necessary if you can just use form.c[0] inline.